app/api/suggestions/specs/configurationManager.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { testingEnvironment } from 'api/utils/testingEnvironment';
+import { SuggestionState } from 'shared/types/suggestionSchema';
+import { createDefaultSuggestions } from '../configurationManager';
+import { IXSuggestionsModel } from '../IXSuggestionsModel';
+import { fixtures, personTemplateId } from './fixtures';
+
+describe('createBlankStates()', () => {
+  beforeEach(async () => {
+    await testingEnvironment.setUp(fixtures);
+  });
+
+  afterAll(async () => {
+    await testingEnvironment.tearDown();
+  });
+
+  it('should create blank states based on settings', async () => {
+    await IXSuggestionsModel.delete({});
+    const settings = [
+      {
+        template: personTemplateId,
+        properties: ['title'],
+      },
+    ];
+
+    await createDefaultSuggestions(settings, 'en', 2);
+    const newSuggestions = await IXSuggestionsModel.get({
+      propertyName: 'title',
+    });
+
+    expect(newSuggestions.length).toBe(2);
+    expect(newSuggestions[0].state).toBe(SuggestionState.valueEmpty);
```

Comment:
Maybe two cases for testing labelEmpty and emptyEmpty as well, as the other two possible blank states? I accept if you think we shouldn't, since technically the calculation is already tested when we are testing the state update.

Reply:
Working on it